tests/conftest.py

```diff
@@ -7,22 +8,16 @@
 collect_ignore = ["test_graphql.py"] if sys.version_info >= (3, 10) else []
 
 
-@pytest.fixture(
-    params=[
```

Comment:
Question: what happened to these params? Where do the values for `anyio_backend_name`/`anyio_backend_options` come from now? (What am I missing?)

Reply:
anyio provides a default set of parameters "trio" and "asyncio"

Reply:
There's another thread here about it https://github.com/encode/starlette/pull/1211#issuecomment-867152021

Reply:
Thanks. So `anyio_backend_options` is going to be `{}` always it looks like?

Reply:
I do think this is a little non-obvious, though, so maybe just a comment with a link to `https://anyio.readthedocs.io/en/stable/testing.html#specifying-the-backends-to-run-on` would be good?